SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -359,4 +365,10 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	// when those properties change.
 	TArray<uint8>* ActorHandoverShadowData;
 	TMap<TWeakObjectPtr<UObject>, TSharedRef<TArray<uint8>>> HandoverShadowDataMap;
+
+	// Band-aid until we get Actor Sets.
+	// Record when an actor got authority from a zoning transition.
+	// If an actor whose position depends on other in the same ownership hierarchy arrives before the others, it could report a position different
+	// from the position it is migrating to. So we prevent the actor from migrating immediately after, we leave some time for the other actors it depends on to arrive.
```

Comment:
I found this comment hard to parse, also added a clarification about this being used only on server-side workers for future readers' clarity.

```suggestion
	// Used on server-side workers only.
	// Record when this worker receives SpatialOS Position component authority over the Actor.
	// Tracking this helps prevent authority thrashing which can happen due a replication race 
	// between hierarchy Actors. This happens because hierarchy Actor migration using the 
	// default load-balancing strategy depends on the position of the hierarchy root Actor. 
	// If the hierarchy Actor data is replicated to the new worker before the hierarchy root, it 
	// can immediately attempt to migrate back. Using this timestamp, we can back off attempting
	// migrations for a while.
```